repoaudit/repoaudit/apt.py

```diff
@@ -48,17 +51,15 @@ def _check_apt_repo_metadata(url: str, dist: str, release_file: Release,
     """Check repo metadata for checksum mismatches."""
 
     dist_url = urljoin(url, "dists", dist)
-    checksum_types = {key: CHECKSUMS[key] for key in CHECKSUMS.keys() if key in release_file}
+    checksums = set(CHECKSUMS.keys()) & set(release_file)
 
     success = True
 
-    files = dict()
-    for key in checksum_types:
+    files = defaultdict(lambda: [])
```

Comment:
Isn't `list` functionally the same as `lambda: []`?

Reply:
Changed to list